plugin/trino-password-authenticators/src/main/java/io/trino/plugin/password/ldap/LdapClient.java

```diff
@@ -17,7 +17,7 @@
 
 import java.util.Set;
 
-public interface LdapAuthenticatorClient
+public interface LdapClient
```

Comment:
I think it has to be part of this commit - https://github.com/trinodb/trino/pull/10116/commits/d3f7eff6991c79e94be1db675aa5fe963374ef8c